README.md

```diff
@@ -187,6 +187,45 @@ Finally,  re-run the project playbook, limiting the scope to the machine that yo
 $ OULIB_USER=vagrant vagrant ssh ansible --command "ansible-playbook /vagrant/projects/web/playbooks/vagrant.yml --limit solr.vagrant.localdomain"
 ```
 
+Ad-hoc Ansible Commands
+------
+As this is an attempt to replicate the production enviornment as closely as possible ad-hoc Ansible commands are not immediately available and require one to SSH into a specific vagrant box within the prod-infrastructure. Below are two scenarios:
```

Comment:
I wonder if this might discourage people a little bit? In my head, a full Ansible environment is immediately available on the control machine, and we save people the bother of doing a local Ansible install and having to worry about versions, etc.